### PR TITLE
Macros for antags

### PR DESCRIPTION
**Pull request by unid15**
Needs reviewing and testing don't merge yet
Will get to it on friday

### CODE REVIEW

**Comment by d3athrow** [comment]
For what purpose.

File: code/__HELPERS/_macros.dm [hunk 1]
```diff
@@ -86,3 +86,31 @@
...
+
+#define ismalf(H) (H.mind in ticker.mode.malf_ai)
+
+#define isnukeop(H) (H.mind in ticker.mode.syndicates)
+
+#define iswizard(H) (H.mind in ticker.mode.wizards)
+
+#define isborer(H) (H.mind in ticker.mode.borers)
+
+#define isbadmonkey(H) ((/datum/disease/jungle_fever in H.viruses) || M.mind in ticker.mode.infected_monkeys)
```
Comment (ComicIronic) [inline code/__HELPERS/_macros.dm]: I am surprised this compiles with no types. Also H and M.

File: code/__HELPERS/_macros.dm [hunk 2]
```diff
@@ -86,3 +86,31 @@
...
+
+#define isdeathsquad(H) (M.mind in ticker.mode.deathsquads)
```
Comment (ComicIronic) [inline code/__HELPERS/_macros.dm]: H and M.

File: code/__HELPERS/_macros.dm [hunk 3]
```diff
@@ -86,3 +86,31 @@
...
 
 #define iswire(O) istype(O, /obj/item/stack/cable_coil)
+
+//Macros for antags
+
+#define isvampire(H) ((H.mind in ticker.mode.vampires) || H.mind.vampire)
+
+#define iscult(H) (H.mind in ticker.mode.cult)
+
+#define isculthead(H) iscult(H)&&(H.mind in ticker.mode.modePlayer)
```
Comment (ComicIronic) [inline code/__HELPERS/_macros.dm]: Should be bracketed

**Comment by Unfit** [comment]
it's to make stuff easier and code prettier i guess

should there be ishuman() or mind checks on the beginning of the macros?

**Comment by ririchiyo** [comment]
I don't like this because it means that people are going to keep using the ticker system instead of moving to a better format for dealing with antagonists.

**Comment by Iamgoofball** [comment]
@ririchiyo why restrict QoL changes based on adding things in the future that don't have any design docs/anyone coding them?

i say :+1:

**Pull request merged**